### DISCUSSION TITLE
How to display the seo title on the Razor page?

### DISCUSSION THREAD

**Discussion by Diman96** [OrchardCMS/OrchardCore#10498] [Q&A]
For liquid page work this `<title>{{ "PageTitle" | shape_new | shape_stringify }}</title>`, but in the documentation I did not find an analogue for Razor pages. 

how to display this data correctly for Razor?
![image](https://user-images.githubusercontent.com/12182810/137448390-aaec0f42-c447-4cfc-ad0b-d02f9c75572a.png)

**Comment by hishamco** [comment]
I think you could access these data from `SeoMetaPartSettings`

**Reply by xperiandri** [reply]
Yes, but in Liquid there is a function that does all the stuff itself and renders all the SEO tags.

**Comment by ns8482e** [comment]
Use `@RenderTitleSegments` from Razor pages / razor views

**Reply by Diman96** [reply]
What exactly do you propose to do? I couldn't find anything in the documentation about `@RenderTitleSegments`

**Comment by ns8482e** [comment]
`@RenderTitleSegments` will add segment  to `<title>` tag

e.g. you could render like this `@RenderTitleSegments("My Page Title")`

Also `@await DisplayAsync(await New.PageTitle())` should work too
